code/game/machinery/transformer.dm

```diff
@@ -10,11 +11,11 @@
 	var/transform_dead = 0
 	var/transform_standing = 0
 	var/cooldown_duration = 600 // 1 minute
```

Comment:
haha what if we took this opportunity to reduce this cooldown duration to six seconds like I did in my stun arm nerf PR wouldn't that cut down on awkwardly waiting around the forceborger haha jk

... unless?

Reply:
😳